README.md

```diff
@@ -62,6 +62,18 @@ $ ant -Dcluster.config=platform
 
 **Note:** You can also use `php`, `enterprise`, etc. See the [cluster.properties](https://github.com/apache/incubator-netbeans/blob/master/nbbuild/cluster.properties) file.
 
+#### Building Windows Launchers
+
+As of [NETBEANS-1145](https://issues.apache.org/jira/browse/NETBEANS-1145), the Windows Launchers can be built adding ```do.build.windows.launchers=true``` property to the build process.
+```
+$ and -Ddo.build.windows.launchers=true
```

Comment:
`ant`

Reply:
Worth mentioning that Windows launchers can be built on other systems. That is very nice!